tests/v2/codecoverage/rancherha/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/corral"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	testSession := session.NewSession()
+
+	corralConfig := corral.CorralConfigurations()
+	err := corral.SetupCorralConfig(corralConfig.CorralConfigVars, corralConfig.CorralConfigUser, corralConfig.CorralSSHPath)
+	if err != nil {
+		logrus.Fatalf("error setting up corral: %v", err)
+	}
+	configPackage := corral.CorralPackagesConfig()
+
+	path := configPackage.CorralPackageImages["aws-rke2-rancher-calico-v1.23.6-rke2r1-2.6.7"]
```

Comment:
What can we do to get a predictable package image name here when creating a corral package with different values.

Like adding a section to `CATTLE_TEST_CONFIG`, generate/update a corral package yaml in the `corral-packages` packages folder and then `build_corral_packages.sh`?

Reply:
I'm not quite sure what you mean. Like different k8s versions and cnis? Would this be a concern? This is just a dummy value. When we update the deployment the latest rancher code coverage image is used.

Reply:
If we don't care about the cluster Rancher is on, this is ok then.